src/components/TasksTable/index.jsx

```diff
@@ -254,6 +315,46 @@ export default class TasksTable extends Component {
             onFilterClick={this.handleFilterToggle}
           />
         </div>
+        <Drawer
+          anchor="right"
+          open={drawerOpen}
+          onClose={this.handleDrawerClose}>
+          <div className={classes.drawer}>
```

Comment:
Can you remove this `div` and put `classes.drawer` on the `Drawer` component directly?